tcms/testruns/static/testruns/js/get.js

```diff
@@ -238,19 +245,26 @@ function renderTestExecutionRow(template, testExecution, testExecutionStatus) {
 /////// the functions below were used in bulk-menu actions
 /////// and need updates before they can be used again
 ///////
-function changeExecutionAssignee() {
-    const executions = 0 // todo: this is the list of all selected executions
-    if (!executions.length) {
-        window.alert(default_messages.alert.no_case_selected);
-        return false;
+function changeAssigneeBulk() {
+    const selected = selectedCheckboxes()
+    if ($.isEmptyObject(selected)) {
+        return false
     }
-    //todo: all texts in alert/prompt must come translated from the HTML template
-    var assignee = window.prompt('Please type new email or username for assignee');
+
+    const enterAssigneeText = $('#test_run_pk').data('trans-enter-assignee-name-or-email')
+    const assignee = prompt(enterAssigneeText)
+
     if (!assignee) {
         return false;
     }
-    executions.forEach(execution_id => jsonRPC('TestExecution.update', [execution_id, { 'assignee': assignee }], () => { }, sync = true));
-    window.location.reload();
+    selected.executionIds.forEach(executionId => {
+        jsonRPC('TestExecution.update', [executionId, { 'assignee': assignee }], () => {
+            const testExecutionRow = $(`div.list-group-item.test-execution-${executionId}`);
+            animate(testExecutionRow, () => {
+                testExecutionRow.find(`.test-execution-asignee`).html(assignee)
```

Comment:
If I type 2 to use an ID then this will result in 2 being shown as the assignee value. Please test more carefully next time.